css/style.css

```
@@ -91,28 +96,110 @@ ul {
     color: var(--basic-white);
 }
 
+.technomart-link {
+    background-color: var(--basic-red);
+    padding: 12px 0 17px 0;
+    height: 18px;
+    flex-basis: 220px;
+}
```

Не хватает внутренней тени
```suggestion
.technomart-link {
    background-color: var(--basic-red);
    padding: 12px 0 17px 0;
    height: 18px;
    flex-basis: 220px;
    box-shadow: inset 0 -5px 0 rgba(0,0,0,.24);
}
```

---

Добавила тень